base/irrationals.jl

```diff
@@ -5,8 +5,9 @@
 """
     AbstractIrrational <: Real
 
-Number type representing an exact irrational value, which is automatically rounded to the correct precision in
-arithmetic operations with other numeric quantities.
+Number type representing an exact irrational value, which is automatically rounded to the "correct" precision in
+arithmetic operations with other numeric quantities. Even the simplest calculations, such as `-pi` or `2π` convert a
+number to `Float64`, making the result no longer irrational.
```

Comment:
Maybe rephrase:

Calculations involving arithmetic between numeric types and irrational types will lead to conversion to a numeric type. For example, -pi or 2π produce `Float64` results.

Reply:
I would like to note that `-pi` is unary, and thus not an arithmetic operation between a numeric type and irrational.

Reply:
Maybe remove it then?

Reply:
I was jus going o change, but I have second thoughts about "Calculations involving arithmetic" (as synonyms?) and "between numeric types". With pi a number, this could be misunderstood, so I was thinking `Number` types, but I see it has misleading help "Abstract supertype for all number types".

Also should clarification rather go into Irrational and/or AbstractIrrational?

Reply:
"-pi is unary, and thus not an arithmetic operation", mathematically not an operation? But in Julia, it's a unary operator as in almost all languages, and involves calculation:

-(x::AbstractIrrational) = -Float64(x)